README.md

```diff
@@ -1,2 +1,39 @@
 # Portfolio
-SergSkol Portfolio
+
+> There is SergSkol Portfolio (Module 1 Day 3)
+
+## Built With
+
+- HTML, CSS
+- Visual Studio Code
+- GIT and GITHUB
+
+## Live Demo (if available)
+
+https://sergskol.github.io/portfolio
+
+## Getting Started
+
+Follow the link to open web page.
+
```

Comment:
- [Optional] Consider adding steps to get a local copy of your project under the getting started section. This will make your work look more professional.

- [Optional] Please in all future projects, make sure that the name of your branch is a feature or an action that is added to the branch not just 'feature'. This is best practice for [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md). A good example of a good branch name is 'headline-section' or 'portfolio-headline'.

- [Optional] Please in all future projects, make sure that your PR title describes the changes implemented in the PR and also your PR description is a summary of all the changes implemented. This will make your work look more professional. Kindly check [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more information.

Reply:
Thank you!